### DISCUSSION TITLE
Is it possible to change split border color?

### DISCUSSION THREAD

**Discussion by J-Siu** [ghostty-org/ghostty#3827] [Q&A] [answered]
If I set background to 000000 (black), split border become barely visible:
![Screenshot 2024-12-28 at 20 20 45](https://github.com/user-attachments/assets/cc3e787c-9c77-4201-a671-3fa0b0db0299)
I want to set it to light grey or white, as the window border.

**Comment by 00-kat** [comment, marked as answer]
Dupe of #3301.
